lib/core/macros.nim

```diff
@@ -509,7 +509,13 @@ proc parseStmt*(s: string): NimNode {.noSideEffect, compileTime.} =
   ## Expects one or more statements. Raises ``ValueError`` for parsing errors.
   result = internalParseStmt(s)
   let x = internalErrorFlag()
-  if x.len > 0: raise newException(ValueError, x)
+  if x.len > 0:
+    var msg = "internalError: "
```

Comment:
How is that an "internalError"?

Reply:
changed to `parse error` (btw, github `insert a suggestion` is quite useful)